lua/ayu/init.lua

```diff
@@ -117,7 +117,7 @@ local function set_groups()
     -- TreeSitter.
     ['@property'] = { fg = colors.tag },
     ['@field'] = { fg = colors.tag },
-    ['@parameter'] = { fg = colors.fg },
+    ['@parameter'] = { fg = colors.parameter },
```

Comment:
Maybe set `@lsp.type.parameter` instead? Because without LSP highlighting parameters doesn't make much sense.

Reply:
just to be sure, do you mean keeping the parameter to be 
```lua
['@parameter'] = { fg = colors.fg },
``` 
and instead of linking `@lsp.type.parameter`, setting it to the color, e.g.  something like?:
```lua
    ['@lsp.type.parameter'] = { fg = colors.lsp_parameter},
```

Setting both parameters allows to keep the same parameter highlighting across the board, even when semantic highlighting is not supported (why I first did it that way). Although this is a gain in consistency for parameters, it could also be confusing since users might expect the parameters to always be colored within the function. Example here in go without semantic highlighting:
![image](https://user-images.githubusercontent.com/7985687/232322989-62f48ced-9fc0-4535-9cd7-74e3bee564af.png)

Reply:
Exactly.

> Although this is a gain in consistency for parameters, it could also be confusing since users might expect the parameters to always be colored within the function

Yes, this is why I think that we should set only `@lsp.type.parameter`.

Reply:
sounds good, I will make the necessary changes